### ISSUE TITLE
Task 4.2: Admin Order UI for Labels and Tracking

### ISSUE THREAD

**Issue by dangoriaynov** [dangoriaynov/speedy_econt_shipping#12]
## Overview
Create admin UI components for viewing, downloading, and printing shipping labels. Display tracking information and shipment status in order details.

## Current State
- Labels generated but no UI to view/download (#11)
- No tracking display in admin
- No way to print labels

### 1. Order Details Meta Box
Add "Shipping Label" meta box to order edit screen (side, high priority):

**When Label Exists:**
- Carrier logo and name
- Tracking number (copyable)
- Creation timestamp
- Download PDF button
- Print button
- Secondary actions: Regenerate, Cancel

**When No Label:**
- "No label generated" message
- "Generate Label" button
- Show validation errors if generation fails

### 2. Tracking Information Section
Add "Shipment Tracking" meta box (normal, default):
- Current status with icon
- Tracking timeline (if API provides)
- "Refresh Tracking" button
- "Track on Carrier Site" external link
- Last updated timestamp

### 3. Orders List Column
Add "Tracking" column after "Status":
- Show tracking number if exists
- Show "—" if no label
- Make tracking number copyable on click

### 4. AJAX Endpoints
```php
wp_ajax_sesh_generate_label     // Generate label for order
wp_ajax_sesh_download_label     // Download PDF
wp_ajax_sesh_print_label        // Print single label
wp_ajax_sesh_bulk_print_labels  // Print multiple labels
wp_ajax_sesh_cancel_label       // Cancel shipment
wp_ajax_sesh_refresh_tracking   // Refresh tracking data
```

### 5. Bulk Actions
Add to orders list bulk actions:
- "Generate Labels" - Create labels for selected orders
- "Print Labels" - Combine and print existing labels

### Meta Box HTML Structure
```html
<div class="sesh-label-box">
  <div class="sesh-label-status">✓ Label Generated</div>
  <div class="sesh-label-info">
    <p><strong>Carrier:</strong> Speedy</p>
    <p><strong>Tracking:</strong> <code>12345678901</code></p>
    <p><strong>Created:</strong> Jan 23, 2026 14:30</p>
  </div>
  <div class="sesh-label-actions">
    <a class="button button-primary">Download PDF</a>
    <a class="button">Print</a>
  </div>
</div>
```

### JavaScript Module
```javascript
// assets/js/sesh-admin-orders.js
SESH_Admin_Orders = {
    generateLabel(orderId),
    downloadLabel(labelId),
    printLabel(labelId),
    bulkPrintLabels(orderIds),
    cancelLabel(labelId),
    refreshTracking(orderId)
}
```

## HPOS Compatibility
Must work with both legacy and HPOS order storage:
```php
// Use SESH_WC_Compat helper (from #15)
SESH_WC_Compat::add_order_meta_box(
    'sesh_shipping_label',
    __('Shipping Label', 'speedy-econt-shipping'),
    [, 'render_label_meta_box'],
    'side',
    'high'
);
```

## Styling
- Use WordPress admin color scheme
- Match WooCommerce order UI design
- Responsive for tablet screens
- Print-friendly CSS for labels

## Acceptance Criteria
- [ ] Meta box shows label status and actions
- [ ] Can download label PDF from order page
- [ ] Can print label directly from admin
- [ ] Print button opens PDF in new window
- [ ] Tracking timeline shows shipment history
- [ ] Orders list shows tracking numbers
- [ ] Bulk print combines multiple labels into one PDF
- [ ] Cancel shipment works and updates status
- [ ] Refresh tracking fetches latest data from API
- [ ] Works with HPOS enabled (#15)
- [ ] All AJAX actions use nonces
- [ ] Errors shown as admin notices

## Dependencies
- **Requires:** #11 (Label Generation) - Must have labels to display
- **Requires:** #14 (Sender Address) - Needed for generation
- **Should integrate:** #15 (HPOS) - Use compatibility helpers
- **Enables:** #13 (Customer Tracking) - Similar UI patterns

## Files to Create
- `includes/admin/class-sesh-admin-orders.php` - Meta boxes and columns
- `includes/admin/class-sesh-admin-ajax.php` - AJAX handlers
- `assets/js/sesh-admin-orders.js` - Frontend logic
- `assets/css/sesh-admin-orders.css` - Styling
- `templates/admin/label-meta-box.php` - Meta box template
- `templates/admin/tracking-meta-box.php` - Tracking template

## Files to Modify
- `includes/admin/class-sesh-admin.php` - Register meta boxes
- `includes/class-sesh-plugin.php` - Load admin classes
